### ISSUE TITLE
Cannot resolve symbol 'res_partner_many2one' in module base

### ISSUE THREAD

**Issue by jcfernandez-890825** [trinhanhngoc/pycharm-odoo#427]
`odoo/addons/base/views/res_partner_views.xml`
![image](https://github.com/odoo-ide/pycharm-odoo/assets/28876647/e5573c88-3492-4758-8d06-d9b5f6eb8b4e)

**Comment by trinhanhngoc** [comment]
@jcfernandez-890825,

Do you think this is a bug of Odoo? The `res_partner_many2one` widget is defined in the `mail` module (https://github.com/odoo/odoo/blob/34825ffadb157cbf37def1ad82d4d523dc199a09/addons/mail/static/src/js/m2x_avatar_user.js#L334) on which the `base` module does not depend.

**Comment by jcfernandez-890825** [comment]
> Do you think this is a bug of Odoo? The `res_partner_many2one` widget is defined in the `mail` module

You are correct, they should move that definition to base or move that XML to another module. But ... that might take a lot of time.

How about this, mail is a module that get's installed as soon as you install any `app` type module. Since it contains all the features to send/fetch emails, chatter on form views, etc ...

Is it possible to consider that this module is a dependency always?
